src/components/molecules/UserProfilePicture/UserProfilePicture.styles.ts

```diff
@@ -35,36 +38,14 @@ export const Reaction = styled.div<ReactionProps>`
     reactionPosition === "right" ? reactionRight : reactionLeft};
   top: -25px;
   // @debt convert this to scss then use our z-index layer helper here
-  z-index: 10;
+  z-index: ${reactionZIndex};
```

Comment:
I believe emoji reactions and shout-outs explicitly had different layers by choice. This change seems to change/break that.